randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2003,7 +2003,10 @@ Extra - asset_id: collision_camera_024
 > Left Room; Heals? False
   * Layers: default
   > Tunnel to Heated U-Turn (bottom)
-      Morph Ball and Varia Suit and Heat/Cold Runs (Intermediate) and Heat Damage ≥ 200
+      All of the following:
+          Morph Ball and Varia Suit
+          Heat/Cold Runs (Intermediate) or Heat Damage ≥ 200
+          Walljump (Beginner) or Simple IBJ or Use Spin Boost
```

Comment:
Varia Suit should not be paired with Morph. It should be in an Or group with the hear run trick.

I want to say beginner wall jump is not enough for the second wall, but i could be wrong. Could you test that out? if it's enough, the last line of options can be removed.

Reply:
You do need 2 wall jumps but it was easily done, not sure where that puts things

Reply:
This would be trivial then if you can just make it with wall jumps. However, you did the Varia Suit thing incorrectly.

Or
-Varia
-And
--Heat/Cold Runs
--Heat Damage